Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/oryx/build:github-actions-20200229.2
+FROM mcr.microsoft.com/oryx/build:github-actions-20200311.3
```

Comment:
Make sure to change the 'target branch' of this PR to `v2-beta`. Currently the target branch is set as `master`